jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -1851,8 +1851,9 @@ ecma_op_object_get_own_property_descriptor (ecma_object_t *object_p, /**< the ob
     }
 
     prop_desc_p->flags |= (JERRY_PROP_IS_VALUE_DEFINED | JERRY_PROP_IS_WRITABLE_DEFINED);
-    prop_desc_p->flags = (uint16_t) (
-      prop_desc_p->flags | (ecma_is_property_writable (property) ? JERRY_PROP_IS_WRITABLE : JERRY_PROP_NO_OPTS));
+    prop_desc_p->flags =
+      (uint16_t) (prop_desc_p->flags
```

Comment:
Do we need the `(prop_desc_p->flags` part?

Reply:
I do not believe so, it can fit into one `|=` expression.